xbmc/windowing/linux/OSScreenSaverFreedesktop.h

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
this was not included in the PR to mainline. Is it something that can be used by other platforms too? Would it make sense?